src/plugins/stake-plugins/thorchainSavers/tcSaversPlugin.tsx

```diff
@@ -1087,6 +1140,220 @@ const unstakeRequestInner = async (opts: EdgeGuiPluginOptions, request: ChangeQu
+  }
+  const thorchainAddresses = await thorchainWallet.getAddresses({ tokenId: null })
+  const thorchainAddress = thorchainAddresses[0].publicAddress
+
+  let memoValue = `tcy:${thorchainAddress}`
+  let memoType: EdgeMemo['type'] = 'text'
+  if (isEvm) {
+    memoType = 'hex'
+    memoValue = base16.stringify(Buffer.from(memoValue, 'utf8'))
+  }
+
+  // Try to spend right out of the primaryAddress
+  const spendInfo: EdgeSpendInfo = {
+    tokenId,
+    spendTargets: [
+      {
+        publicAddress: poolAddress,
+        nativeAmount: nativeAmount
+      }
+    ],
+    memos: [
+      {
+        type: memoType,
+        value: memoValue
+      }
+    ],
+
+    // Use otherParams to meet Thorchain Savers requirements
+    // 1. Sort the outputs by how they are sent to makeSpend making the target output the 1st, change 2nd
+    // 2. Only use UTXOs from the primary address (index 0)
+    // 3. Force change to go to the primary address
+    otherParams: {
+      enableRbf: false,
+      outputSort: 'targets',
+      utxoSourceAddress,
+      forceChangeAddress
+    },
+    assetAction: { assetActionType: 'claim' },
+    savedAction: {
+      actionType: 'stake',
+      pluginId: stakeProviderInfo.pluginId,
+      stakeAssets: [
+        {
+          pluginId,
+          tokenId,
+          nativeAmount
+        }
+      ]
+    }
+  }
+
+  if (isEvm) {
+    // For mainnet coins of EVM chains, use gasLimit override since makeSpend doesn't
+    // know how to estimate an ETH spend with extra data
+    spendInfo.networkFeeOption = 'custom'
+    spendInfo.customNetworkFee = {
+      ...spendInfo.customNetworkFee,
+      gasLimit: EVM_SEND_GAS
+    }
+  }
+
+  let networkFee = '0'
+  let needsFundingPrimary = false
+  if (lt(addressBalance, nativeAmount)) {
+    // Easy check to see if primary address doesn't have enough funds
+    if (isEvm) {
+      // EVM chains only have one address, so if there aren't enough funds in
+      // the primary address then we don't have enough funds at all
+      throw new InsufficientFundsError({ tokenId: null })
+    }
+    needsFundingPrimary = true
+  } else {
+    try {
+      const estimateTx = await wallet.makeSpend(spendInfo)
+      networkFee = estimateTx.parentNetworkFee ?? estimateTx.networkFee
+    } catch (e: unknown) {
+      if (!isEvm && asMaybeInsufficientFundsError(e) != null) {
+        needsFundingPrimary = true
+      } else {
+        throw e
+      }
+    }
+  }
+
+  let fundingSpendInfo: EdgeSpendInfo | undefined
+  if (needsFundingPrimary) {
+    // Estimate the total cost to create the two transactions
+    // 1. Fund the primary address with the requestedAmount + fees for tx #2
+    // 2. Send the requested amount to the pool address
+    fundingSpendInfo = {
+      tokenId: null,
+      spendTargets: [
+        {
+          publicAddress: primaryAddress,
+          nativeAmount: nativeAmount
+        }
+      ],
+      assetAction: { assetActionType: 'stakeNetworkFee' },
```

Comment:
This should be `claimNetworkFee` but we don't have that. Either add to core or just use `unstakeNetworkFee`